app/views/staff/tracks/update.js.erb

```diff
@@ -1,4 +1,10 @@
-$('#config-modal').modal('hide');
+<% if @track.save %>
```

Comment:
Is it your intention to save the track record here? I think that was done in the controller already. I would say go with @track.errors.present?. If the save didn't work out, there'll be errors in that collection, but on success it'll be empty.